### PR TITLE
feat: Add sound support

### PR DESCRIPTION
**Pull request by dadav**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/PhotoboothProject/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)

- [ ] Documentation update
- [ ] Bug fix
- [x] New feature
- [ ] Other, please explain:

#### What changes did you make? (Give an overview)

Added some basic sounds and the required code. But it's not working yet.

See #310

### CODE REVIEW

File: resources/lang/en.json
Comment (andi34) [inline resources/lang/en.json:1]: https://github.com/andi34/i18n-editor-release

Should be edited using the i18n editior

File: docs/configuration/sounds.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Sounds
+
+If you have speakers connected to your photobox, you can enable the sound
```
Comment (andi34) [inline docs/configuration/sounds.md]: Photobooth instead of photobox

**Comment by dadav** [comment]
Summary:
- Add sounds for countdown + cheese
- Sounds are disable by default
- Every group of sounds is deactivateable (countdown, cheese)
- The english sounds are used as a fallback (can be deactivated)
- Custom sounds can be added in `/private/sounds`

File: resources/lang/de.json
Comment (andi34) [inline resources/lang/de.json:1]: Need to be added via Crowdin.

**Review by andi34 — CHANGES_REQUESTED** [review]

File: resources/lang/en.json
Comment (dadav) [reply resources/lang/en.json:1]: Ok, I've used the editor now.

File: resources/lang/de.json
Comment (dadav) [reply resources/lang/de.json:1]: Okay, I've removed my changes.

File: resources/lang/en.json
Comment (andi34) [inline resources/lang/en.json:1]: There's a lot strings removed, by accident I guess

**Review by andi34 — CHANGES_REQUESTED** [review]

File: resources/lang/en.json
Comment (dadav) [reply resources/lang/en.json:1]: Hm, I've used the editor, don't know what went wrong

Comment (dadav) [reply resources/lang/en.json:1]: I opened the project and saved it again, so it would adjust the formatting.

Comment (andi34) [reply resources/lang/en.json:1]: Mh... There's a lot of manual entries removed which shouldn't. Some of the formatting earlier broke and removed it?

Comment (dadav) [reply resources/lang/en.json:1]: I think now it should be fine. It somehow didnt handle the manual additions well.

**Review by andi34 — APPROVED** [review]

**Pull request merged**